src/test/emulators/extensions/postinstall.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { expect } from "chai";
+import * as Sinon from "sinon";
+import * as postinstall from "../../../emulator/extensions/postinstall";
+import { EmulatorRegistry } from "../../../emulator/registry";
+import { Emulators } from "../../../emulator/types";
+
+describe("replaceConsoleLinks", () => {
+  let sandbox: Sinon.SinonSandbox;
+  beforeEach(() => {
+    sandbox = Sinon.createSandbox();
+    sandbox
+      .stub(EmulatorRegistry, "getInfo")
+      .returns({ name: Emulators.UI, host: "localhost", port: 4000 });
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  const tests: {
+    desc: string;
+    input: string;
+    expected: string;
+  }[] = [
+    {
+      desc: "should replace Firestore links",
+      input:
+        " Go to your [Cloud Firestore dashboard](https://console.firebase.google.com/project/test-project/firestore/data) in the Firebase console.",
```

Comment:
another common pattern is 
`https://console.firebase.google.com/u/0/project/test-project/firestore/data` if there's multiple GAIA users logged in. Can you add a test case for that?